auth/src/main/java/com/firebase/ui/auth/data/model/FlowParameters.java

```diff
@@ -128,4 +131,24 @@ public FlowParameters createFromParcel(Parcel in) {
             return new FlowParameters[size];
         }
     };
+
+    public boolean isSingleProviderFlow() {
+        return providerInfo.size() == 1;
+    }
+
+    public @StringRes int getGlobalTermsStringResource() {
+        boolean hasTos = !TextUtils.isEmpty(termsOfServiceUrl);
+        boolean hasPp = !TextUtils.isEmpty(privacyPolicyUrl);
+        if(hasTos && hasPp) {
+            return R.string.fui_tos_and_pp;
+        } else if(hasTos) {
+            return R.string.fui_tos_only;
+        } else {
+            return R.string.fui_pp_only;
+        }
+    }
+
+    public @StringRes int getGlobalTermsFooterStringResource() {
+        return R.string.fui_tos_and_pp_footer;
```

Comment:
Should this have if statements like above? If not, let's just inline it in the XML.

Reply:
I added 2 more strings to deal with ToS only & Pp only. Having this makes more sense now :)